node/perf-test/src/lib.rs

```diff
@@ -44,4 +44,17 @@ pub struct PerfCmd {
 
 	#[structopt(long, value_name = "CHAIN_SPEC", default_value = "dev")]
 	pub chain: String,
+
+	#[structopt(
+		long = "disable-sysinfo",
+		help = "Do not attempt to query system info."
+	)]
+	pub disable_sysinfo: bool,
+
+	// TODO: allow multiple tests (like the -l switch for logging)
+	#[structopt(
+		long = "tests",
+		help = "Comma-separated list of tests to run (if omitted, runs all tests)"
+	)]
+	pub tests: Option<String>,
```

Comment:
```suggestion
	pub tests: Vec<String>,
```
This will allow you to do `--tests test1 --tests test2 --tests test3` which is not so bad if you give it a short option like `-t`. I don't know how to get the syntax you described though.

Reply:
I'm not sure why this showed up in this PR, it was introduced in a different one.

Anyway, the `Vector<...>` trick sounds useful, but I added code to parse a comma-separated list (in that other PR) as the TODO suggested -- so I removed the TODO comment.